client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -139,6 +139,35 @@ public void shouldNotRetryFailedPostCalls() {
         }
     }
 
+    @Test
+    public void shouldNotRetryFailedPostCallsWithSystemExceptions() {
+        AtomicLong callCount = new AtomicLong();
+        HttpServer<ByteBuf, ByteBuf> server = startServer(OK, "\"TAKES LONGER THAN READ TIMEOUT\"", r -> {
+            callCount.incrementAndGet();
+            try {
+                Thread.sleep(6);
```

Comment:
hehe, that's blocking code, dude! Search for other tests in this class "Slow server" to find examples of a proper slow server.